terraform/deployments/github/repos.yml

```
@@ -271,6 +271,11 @@ repos:
     need_production_access_to_merge: false
     allow_squash_merge: true
     push_allowances: []
+  
+  govuk-display-screen:
+    need_production_access_to_merge: false
+    allow_squash_merge: true
+    standard_contexts: *standard_security_checks
```

To quickly give people in this group access to the repo we can add the following config:
```suggestion
    standard_contexts: *standard_security_checks
    teams: {
      govuk: "maintain"
      }
```

Note that is gives access (read, create and manage pull requests + some repo settings) to everyone in the GOV.UK team, which includes other people. See a list [here](https://github.com/alphagov/govuk-user-reviewer/blob/main/config/govuk_non_tech.yml) (private repo). This is the approach we took for  https://github.com/alphagov/govuk-design-guide repo.

To give permissions to only people in the [GOV.UK Data team](https://github.com/orgs/alphagov/teams/gov-uk-data) would require importing that team into terraform and updating some config logic, which takes a bit more time than I have at the moment.

Please let me know if there’re any issues with this approach, otherwise we'd be happy to add the work to our backlog.